dagmc_stats.py

```diff
@@ -153,51 +154,111 @@ def get_surfaces_per_volume(my_core, entityset_ranges):
+    tris : (list)triangles
     """
-
     if my_core.tag_get_data(geom_dim, meshset)[0][0] == 3:
         entities = my_core.create_meshset()
         for surface in my_core.get_child_meshsets(meshset):
             my_core.add_entities(entities, my_core.get_entities_by_type(surface, types.MBTRI))
         tris = my_core.get_entities_by_type(entities, types.MBTRI)
+    elif my_core.tag_get_data(geom_dim, meshset)[0][0] == 2:
+        entities = my_core.create_meshset()
+        my_core.add_entities(entities, my_core.get_entities_by_type(meshset, types.MBTRI))
+        tris = my_core.get_entities_by_type(entities, types.MBTRI)
     else:
         tris = my_core.get_entities_by_type(meshset, types.MBTRI)
+    return tris
 
-    t_a_r = []
 
-    for triangle in tris:
-        side_lengths = []
-        s = 0
-        coord_list = []
+def get_tri_side_length(my_core, tri):
+    """
+    get side lengths of triangle
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    tri : triangle
+
+    outputs
+    -------
+    side_lengths : (list)side lengths of triangle
+    """
+    side_lengths = []
+    s = 0
+    coord_list = []
+
+    verts = list(my_core.get_adjacencies(tri, 0))
+
+    for vert in verts:
+        coords = my_core.get_coords(vert)
+        coord_list.append(coords)
 
-        verts = list(my_core.get_adjacencies(triangle, 0))
+    for side in range(3):
+        side_lengths.append(np.linalg.norm(coord_list[side]-coord_list[side-2]))
+        # The indices of coord_list includes the "-2" because this way each side will be matched up with both
+        # other sides of the triangle (IDs: (Side 0, Side 1), (Side 1, Side 2), (Side 2, Side 0))
+    return side_lengths
 
-        for vert in verts:
-            coords = my_core.get_coords(vert)
-            coord_list.append(coords)
 
-        for side in range(3):    
-            side_lengths.append(np.linalg.norm(coord_list[side]-coord_list[side-2]))
-            # The indices of coord_list includes the "-2" because this way each side will be matched up with both
-            # other sides of the triangle (IDs: (Side 0, Side 1), (Side 1, Side 2), (Side 2, Side 0))
-      
+def get_triangle_aspect_ratio(my_core, meshset, geom_dim):
+    """
+    Gets the triangle aspect ratio (according to the equation: (abc)/(8(s-a)(s-b)(s-c)), where s = .5(a+b+c).)
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    meshset : a meshset containing a certain part of the mesh
+    geom_dim : a MOAB Tag that holds the dimension of an entity.
+
+    outputs
+    -------
+    t_a_r : (list) the triangle aspect ratios for all triangles in the meshset
+    """
+    tris = get_tris(my_core, meshset, geom_dim)
+    t_a_r = []
+
+    for tri in tris:
+        side_lengths = get_tri_side_length(my_core, tri)
         s = .5*(sum(side_lengths))
         top = np.prod(side_lengths)
         bottom = 8*np.prod(s-side_lengths)
         t_a_r.append(top/bottom)
```

Comment:
These lines here could be moved into a function `calc_tri_aspect_ratio(side_lengths)` like @gonuke suggested. Then it could be called here and it would be easier to test.

@gonuke correct me if this is not what you're suggesting.